convert2rhel/actions/pre_ponr_changes/handle_packages.py

```diff
@@ -132,6 +134,12 @@ def run(self):
             # There is needed for avoid reaching out RHEL repositories while requesting info about pkgs.
             disable_repos = repo.get_rhel_repos_to_disable()
             pkgs_removed = _remove_packages_unless_from_redhat(pkgs_list=all_pkgs, disable_repos=disable_repos)
+
+            # RHELC-1677
+            # Some repofile packages can remove /etc/yum.repos.d/ when they are removed. Subscription-manager
+            # doesn't expect this and without the directory the redhat.repo isn't created. This results in unability
+            # to access repositories as the repository directory doesn't exist.
+            _check_repos_directory()
```

Comment:
I would rename this as it is not only checking, but actually creating a folder in case it is not there anymore. 

If it was strictly checking and giving us a response based on that, the naming would be fine, otherwise, since we are creating it, then the name does not apply anymore.

Reply:
Good point, thanks! I renamed it to `fix_repos_direcotory`. I was careful with the word "create" as we are not creating something new, but "fixing" something broken.